configs/resetPasswEmail.tmpl

```diff
@@ -0,0 +1,7 @@
+To: {{.To}}
+From: {{.From}}
+Subject: {{.Subject}}
+{{.Header}}
+{{.Content}}
```

Comment:
Can you move the content here? 
Idea was to use a template's because we don't want cont's with text in code, We want the static text in template's. 
Move 
```
const (
	message = `
You have initiated password reset.
Follow the link below to reset password.`
)
```
Here and pass reset URL as a variable in the template.

Reply:
Also, I don't like the location of templates `config/<template>` 
I would put it in `/users/emailer/templates` and call `mailer` not `emailer`, details ;)

Reply:
mailer was my initial changed to emailer per @manuio request